src/main/java/uk/gov/di/ipv/cri/common/library/helper/LogHelper.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.di.ipv.cri.common.library.helper;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.cloudwatchlogs.emf.util.StringUtils;
+import software.amazon.lambda.powertools.logging.LoggingUtils;
+import uk.gov.di.ipv.cri.common.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+public class LogHelper {
```

Comment:
Can the necessary functionality from this class be added to the existing `EventProbe` class ?
Furthermore, all passport (CRI) specific code should be removed from here and field names etc should simply be passed in

Reply:
@gds-jbentham Okay, that's fair enough.
If this class is just used by passport CRI can this class reside in the passport repo ?
( rather than common )

Reply:
The issue is were planning to move the log helper lines into the common lambdas so theres no change in existing functionality / logging when passport is switched. If we limit it to passport when we switch the lambdas over passport potentially lose logging. There will be an action afterwards possibly a short call on what we want to log going forward across all the CRIs

I have another PR to raise after this which adds logging helper to the session and access token lambdas